torch/csrc/jit/script/tree_views.h

```diff
@@ -260,10 +260,10 @@ struct Attribute : public TreeView {
   Ident name() const {
     return Ident(subtree(0));
   }
-  Expr value() const {
-    return Expr(subtree(1));
+  TreeRef value() const {
+    return subtree(1);
   }
-  static Attribute create(const SourceRange& range, const Ident& name, const Expr& value) {
+  static Attribute create(const SourceRange& range, const Ident& name, const TreeRef& value) {
```

Comment:
Hey, what's going on in here? We shouldn't be doing any funny business with the AST... Let's keep it statically typed, with the grammar at the top of the file updated... If you want to change it then fine, but keep the types!!